packages/api/src/orbitAPI.ts

```diff
@@ -89,6 +75,41 @@ export type Spec = {
    */
 };
 
+export type ValidatableSpec = {
+  "/actionLogs"?: {
+    GET?: {
+      query: {
+        /**
+         * @minimum 1
+         * @default 100
+         * @TJS-type integer
+         */
+        limit?: number;
+        /**
+         * Created After ActionLogID
+         * @TJS-type string
```

Comment:
Idle wondering: is it possible to add this annotation to the definition of `ActionLogID` so that it propagates to all client sites, rather than needing to repeat at each one?

Reply:
Good idea, I actually did not consider this beforehand. Unfortunately, it looks `typescript-json-schema` is not cooperating. In theory, what you mentioned should work, however `ActionLogID` is defined as a `type` (rightly so), but it looks like the library does not use the comment for it. I also tried creating a _box_ of sorts:

```typescript
interface BoxActionLogID {
  id: string & { __actionLogIDOpaqueType: never };
}
export type ActionLogID = BoxActionLogID["id"];
```

This works! But, only when the id is not defined as an array, so still not great. I believe this to be a bug. If I have sometime soon, I'll see if I can create an issue on their end.